src/odemis/gui/util/img.py

```diff
@@ -1611,9 +1616,16 @@ def get_ordered_images(streams, raw=False):
         # FluoStreams are merged using the "Screen" method that handles colour
         # merging without decreasing the intensity.
         if not raw:
-            data = s.image.value
+            if isinstance(s.raw, list):
+                data = s.image.value
+            else:
+                data = img.mergeTiles(s.image.value)
         else:
-            data_raw = s.raw[0]
+            if isinstance(s.raw, list):
```

Comment:
Same thing here, first get data_raw, and then if it's a 2D tuple call mergeTiles() on it. In particular, you cannot call ._getMergedRawImage() as it's private.  Also, this will avoid loading the entire file to export just the current view.